tf_patches/gpu_hlo_schedule.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/tensorflow/compiler/xla/service/gpu/gpu_hlo_schedule.cc b/tensorflow/compiler/xla/service/gpu/gpu_hlo_schedule.cc
+index 1197bc8b323..b4809b50e4f 100644
+--- a/tensorflow/compiler/xla/service/gpu/gpu_hlo_schedule.cc
++++ b/tensorflow/compiler/xla/service/gpu/gpu_hlo_schedule.cc
+@@ -356,7 +356,7 @@ Status ScheduleGpuModule(HloModule* module, int64_t pointer_size,
+       HloPrintOptions::Canonical().set_print_backend_config(true));
+   HloInstruction* root = module->entry_computation()->root_instruction();
+   FrontendAttributes attributes;
+-  (*attributes.mutable_map())[kFingerprintBeforeLHS] = fingerprint;
++  (*attributes.mutable_map())[std::string(kFingerprintBeforeLHS)] = fingerprint;
```

Comment:
can we submit a cl to fix this internally too?

Reply:
cl has been created.